farmdata2_modules/fd2_tabs/resources/CustomTableComponent.spec.comp.js

```diff
@@ -505,4 +505,54 @@ describe('custom table component', () => {
             })
         })
     })
+
+    context('export csv file', () => {
+        const path = require("path");
+        const downloadsFolder = 'cypress/downloads'
+        let comp;
+        beforeEach(() => {
+            comp = mount(CustomTableComponent, {
+                propsData: {
+                    rows: [ {id: 10, data: [12, 3, 'answome']},
+                    {id: 11, data: [19, 3, 'and'],},
+                    {id: 12, data: [12, 12, 'answome12'],}, 
+                    ],
+                    headers: ['cool', 'works?', 'hello'],
+                    canDelete: true,
+                    visibleColumns: [true, true, false],
+                }
+            })
+        })
+
+        it('assure the button exists', () => {
+            cy.get('[data-cy=export-btn')
+            .should('have.exist')
+        })
+
+        it('verifies download', () => {
+            var today = new Date();
+            var dd = String(today.getDate()).padStart(2, '0');
+            var mm = String(today.getMonth() + 1).padStart(2, '0'); 
+            var yyyy = today.getFullYear();
+            today = mm + dd + yyyy;
+            cy.get('[data-cy=export-btn]')
+                .click();
+
+            cy.readFile(path.join(downloadsFolder, 'seedingReport_' + today + '.csv')).should("exist");
+
+        });
+
+        it('verifies content of csv', () => {
+            var today = new Date();
+            var dd = String(today.getDate()).padStart(2, '0');
+            var mm = String(today.getMonth() + 1).padStart(2, '0'); 
+            var yyyy = today.getFullYear();
+            today = mm + dd + yyyy;
+            cy.get('[data-cy=export-btn]')
+                .click();
+
+            cy.readFile(path.join(downloadsFolder, 'seedingReport_' + today + '.csv')).should("eq", "cool,works?\n12,3\n19,3\n12,12");
+            cy.exec("cd " + downloadsFolder + ' ' + "&& rm -rf *")
```

Comment:
This looks like it deletes the contents of the downloads folder, but not the folder itself.  Is the actual downloads folder also deleted?

Reply:
Let me test something real quick and get back to you on this.

Reply:
> This looks like it deletes the contents of the downloads folder, but not the folder itself. Is the actual downloads folder also deleted?

I just pushed a change. I was unsure if Cypress was creating the folder, but I now have a better understanding of the code. Cypress is still creating the folder and now it also deletes it since its the owner of the folder. 
This new change deletes the folder all together after the tests.